protokube/pkg/protokube/do_volume.go

```diff
@@ -259,6 +314,13 @@ func getMetadataDropletID() (string, error) {
 	return getMetadata(dropletIDMetadataURL)
 }
 
+func getMetadataDropletTags() ([]string, error) {
+
+	tagString, error := getMetadata(dropletIDMetadataTags)
+	dropletTags := strings.Split(tagString, "\n")
+	return dropletTags, error
```

Comment:
you could combine these two rows as `return strings.Split(tagString, "\n"), err`